src/store/services/calendar/index.ts

```diff
@@ -44,8 +44,7 @@ export const googleCalendarApi = createApi({
       { id: string; startDate: string; endDate: string }[],
       string | void
     >({
-      query: (mail) =>
-        "/api/calendar/get-appointment-by-mail" + (mail ? `?mail=${mail}` : ""),
+      query: () => "/api/calendar/get-appointment-by-mail",
```

Comment:
mmmmm... così non mi stai più passando la mail nei params, non funziona più in prod. 
Deve essere `query: mail =>  "/api/calendar/get-appointment-by-mail?email=${mail}"` 
Il params nell'URL deve essere "?email"